create-release.sh

```diff
@@ -24,9 +24,10 @@ tar -czf ../$FILENAME *
 cd ..
 
 
-# Build version for root path installation
+# Build integrated version
 rm -rf build/
 export PUBLIC_URL=/studio
+export REACT_APP_SETTINGS_PATH="../ui/config/studio/settings.json"
```

Comment:
Will this still work if you access Studio via e.g. https://opencast.example.com/studio/about ?

```suggestion
export REACT_APP_SETTINGS_PATH="/ui/config/studio/settings.json"
```

Reply:
Good catch. I mean currently, this URL will lead to a 404 if visited directly. Opencast does not yet implement that single page app forwarding, as you know. But yes, your suggestion is the proper solution.